parser/V22/parserImpl_V22.go

```diff
@@ -225,7 +225,7 @@ func (p *Parser) parseTrace(trace typesv22.ExecutionTraceV22, mainMsgCid cid.Cid
 		BasicBlockData: types.BasicBlockData{
 			Height:    uint64(tipset.Height()),
 			TipsetCid: tipsetCid,
-			BlockCid:  blockCid,
+			BlocksCid: []string{blockCid},
```

Comment:
Are we sure that by doing this the result in the db will be ["sadsad","sadsad","sadsad","sadsad"] instead of "{sadsad, sadsad, sadsad, sadsad}? I am actually not sure. Maybe blockCid is already a string, and the new result will be ["{sadsad, sadsad, sadsad, sadsad}"].... can we check please?

Reply:
I fixed Emma!